messenger.go

```diff
@@ -690,6 +699,26 @@ func (m *Messenger) RetrieveRawAll() (map[transport.Filter][]*protocol.StatusMes
 	return result, nil
 }
 
+func buildContact(publicKey *ecdsa.PublicKey) (*Contact, error) {
+	address := strings.ToLower(crypto.PubkeyToAddress(*publicKey).Hex())
+
+	id := fmt.Sprintf("0x%s", hex.EncodeToString(crypto.FromECDSAPub(publicKey)))
```

Comment:
```suggestion
	id := "0x" + hex.EncodeToString(crypto.FromECDSAPub(publicKey))
```

Or this new `hexBytes` struct.